frontend/src/components/projectDetailsBody/ProjectDetailsBody.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import './ProjectDetailsBody.css';
+import { Checkbox } from '@mui/material';
+import {
+  CheckCircleRounded,
+  AccountCircleRounded,
+  CalendarMonthRounded,
+  FlagRounded,
+  CheckCircle,
+  RadioButtonUnchecked,
+  Flag,
+} from '@mui/icons-material/';
+
+import { Task } from '../../pages/projectView/ProjectView';
+
+type ProjectDetailsBodyType = {
+  task: Task;
+};
+
+const ProjectDetailsBody = ({ task }: ProjectDetailsBodyType) => {
+  return (
+    <div className='details-title'>
+      <div className='project-name'>
+        <Checkbox
+          readOnly
+          checked={task.completed}
+          icon={<RadioButtonUnchecked style={{ color: '#7D7A89' }} />}
+          checkedIcon={<CheckCircle style={{ color: '#5FB918' }} />}
+        />
+        <div className='label'>
```

Comment:
The reason for that repeatition is that I splited ProjectDetails into 3 components:
1- ProjectDetailHeader
2-ProjectDetailBody
3-ProjectDetailRest (I admit this is not a good naming!)

and then I passe the 'task' prop from parent ProjectDetail to ProjectDetailBody child.
but if you think this approach will lead to a code repetition I will put all the code for ProjectDetailBody directly into the ProjectDetail.

![Screenshot 2023-10-03 at 18 49 46](https://github.com/HackYourFuture-CPH/class-25-lothbrok/assets/71033948/467557da-d271-4eec-8c85-9a3c6437d3fa)

Reply:
I think my confusion is this:
You have the concept of Projects
Projects can have a list of tasks

I do not get why the Project component need to have one task passed in instead of a Project object that has many tasks.